lms/services/blackboard_api/_schemas.py

```diff
@@ -57,14 +58,23 @@ def post_load(self, data, **_kwargs):  # pylint:disable=no-self-use
         return data["results"]
 
 
-class BlackboardListGroupSetGroups(RequestsResponseSchema):
+class BlackboardListGroups(RequestsResponseSchema):
```

Comment:
Renamed the schema as well, this is the schema of a list of groups that we might use when listing the groups of a groupset, the groups of a course, etc